src/main/java/lotto/domain/Lotto.java

```diff
@@ -0,0 +1,88 @@
+package lotto.domain;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
+public class Lotto {
+    private final List<LottoNumber> numbers;
+
+    public Lotto(int... numbers) {
+        this(createNumbers(intToList(numbers)));
+    }
+
+    public Lotto(String... numbers) {
+        this(createNumbers(stringToList(numbers)));
+    }
+
+    public Lotto(List<LottoNumber> numbers) {
+        this.numbers = numbers;
+    }
+
+    public static Lotto of(List<Integer> numbers) {
+        return new Lotto(createNumbers(numbers));
+    }
+
+    private static List<LottoNumber> createNumbers(List<Integer> numbers) {
+        List<LottoNumber> list = new ArrayList<>();
+        for (Integer number : numbers) {
+            list.add(new LottoNumber(number));
+        }
+        return list;
+    }
+
+    private static List<Integer> stringToList(String... numbers) {
+        List<Integer> list = new ArrayList<>();
+        for (String number : numbers) {
+            list.add(Integer.parseInt(number));
+        }
+        return list;
+    }
```

Comment:
```suggestion
    private static List<LottoNumber> stringToList(String... numbers) {
        List<LottoNumber> list = new ArrayList<>();
        for (String number : numbers) {
            list.add(new LottoNumber(number));
        }
        return list;
    }
```
이와 같이 구현해도 되지 않을까?
이미 일정 수준의 객체 설계 역량이 쌓였으니 stream api로 도전해 보는 것도 추천
intToList() 메서드도 같은 피드백